src/components/20-molecules/image-upload/README.md

```diff
@@ -69,37 +69,37 @@ Import the image-upload-defining script and use a image-upload like this:
 
 ### Variant
 
-| Attribute                                  | Details                                                              |
-| ------------------------------------------ | -------------------------------------------------------------------- |
-| `inputFileText="Upload file"`              | Text in the input-file component                                     |
-| `maxSizeOfSingleFileKB="10"`               | Maximal size of a single file in KB                                  |
-| `maxSizeOfAllFilesKB="15"`                 | Maximal size of a all files together in KB                           |
-| `maxNumberOfFiles="10"`                    | Maximal number of files                                              |
-| `showImageOverview="false"`                | Switch between normal view and image overview                        |
-| `icon="cloud-upload"`                      | Specify the upload icon in input-file component                      |
-| `fileTooBigStatusText="Error occurred"`    | Specify the error message for to big files                           |
-| `tooManyFilesStatusText="Error occurred"`  | Specify the error message for to many files                          |
-| `filesTooBigStatusText="Error occurred"`   | Specify the error message when maximal size of all files is exceeded |
-| `deleteStatusText="Delete"`                | Specify the caption when hovering over a file                        |
-| `addStatusText="Add more"`                 | Specify the caption of dashed box                                    |
-| `infoText="drag and drop your files here"` | Specify the text on the start view                                   |
-| `orText="or"`                              | Specify the text on the start view box                               |
+| Attribute                                  | Details                                                               |
+| ------------------------------------------ | --------------------------------------------------------------------- |
+| `inputFileText="Upload file"`              | Text in the input-file component                                      |
+| `maxSizeOfSingleFileKB="10"`               | Maximal size of a single file in KB                                   |
+| `maxSizeOfAllFilesKB="15"`                 | Maximal size of a all files together in KB                            |
+| `maxNumberOfFiles="10"`                    | Maximal number of files                                               |
+| `showImageOverview="false"`                | Switch between normal view and image overview                         |
```

Comment:
would it make sense to call it file overview? Bc we are renaming the image-upload anyway. And technically it does show images and files

Reply:
Yes rename that stuff. But that has to go with the change in the code, not that code and readme diverges.